screens/ProductionScreens/ProductionDashboard.jsx

```diff
@@ -103,6 +105,20 @@ export default function ProductionDashboardScreen({ route }) {
                     </View>
                 </>
             )}
+            <TouchableOpacity
```

Comment:
test alerts can go right? unless you're still using them until we get proper production dashboard functionality

Reply:
Was thinking keep, but have removed as we have an example of usage in UserDetailsModal